nav2_system_tests/src/system/test_multi_robot_launch.py

```diff
@@ -52,22 +52,22 @@ def generate_launch_description():
 
     # Launch Gazebo server for simulation
     start_gazebo_cmd = ExecuteProcess(
-            cmd=['gzserver', '-s', 'libgazebo_ros_factory.so',
-                 '--minimal_comms', world],
+            cmd=['gzserver', '-s', 'libgazebo_ros_init.so',
```

Comment:
@SteveMacenski  this change fixes the multi_robot system test. with few minor changes and i managed to make the test passed locally. shall i commit  the changes regarding to it also in this PR?

Reply:
I'm not sure what you mean? Are these changes needed before this PR can go in? Maybe it would make sense for me to approve and merge this and then you submit a new PR with those proposed changes that we can look at. That way we can get this work completed and discuss that separately if there's any concerns.

Reply:
@SteveMacenski  i will open new pr on it later.
 
i cannot think of anything else to do in this pr. i tried the changed launch files on foxy, galactic , rolling. nav2  succeeded on all.  there was some error on rviz2 material rendering in rolling. 
![Screenshot from 2021-08-19 06-30-39](https://user-images.githubusercontent.com/57533054/129991630-6fd74a46-15c6-47a2-9127-3a393c1cf81b.png).
i don't think it is related to this pr.

you can merge this. if you are okay with it.

thanks. it's my pleasure to work with you on my first PR. i am eager to learn more  and contribute.

Reply:
It was very much so appreciated, thanks for your hard work! Another thing in the follow up PR might be to remove the `bringup` directory (e.g. we have `nav2_bringup/bringup/<pkg stuff>` since we no longer the nav2_gazebo_spawner in that directory. 

Let me know if you're interested in working on another ticket / project! I'd definitely appreciate it :smile:

Reply:
sure.
i thought to do it in this pr . since  I'm  new to github things i was feared that doing it might mess things up.  i will create a follow up PR. more PR's will make my github profile look good.

I am not a CS major . i have completed post graduate in mechanical design.  Robotics and ROS made me interested towards programming. i love to learn and work in c++. i am currently free i will find another project that is suitable to me soon most probably in nav2.